### ISSUE TITLE
Update instaflux demo with media capabilities when MediaStore is in develop

### ISSUE THREAD

**Issue by tonyr59h** [wordpress-mobile/WordPress-FluxC-Android#123]
instaflux was originally intended to upload images but the MediaStore is under development so text posts were implemented instead. After #122 is merged instaflux can be updated to demonstrate the MediaStore.

**Comment by oguzkocer** [comment]
I think this is already done as Instaflux is actually creating media posts right now #252 and `MediaStore` has been merged to `develop`. So, I think we can close this one.
